app/services/sort_stories.rb

```diff
@@ -0,0 +1,25 @@
+class SortStories
+  attr_reader :stories
```

Comment:
You can remove this or do `stories.each do |story|` [here](https://github.com/Codeminer42/cm42-central/pull/306/files#diff-db329cd1427b4a7d53fa925fa7683663R10)

I'd prefer to remove this `attr_reader` since you just use `stories` once.